### DISCUSSION TITLE
[중요]트랙차트 컴포넌트 추가 및 큐레이션, 아티스트, 포스트 컴포넌트 수정

### DISCUSSION THREAD

**Discussion by ryan-g00** [prgrms-fe-devcourse/NFE1_2_2_MUX#74] [D. 아이디어&레퍼런스]
![image](https://github.com/user-attachments/assets/dccbb337-3194-4968-98cf-fb71ac003bdc)
컴포넌트 작업을 하다보니까.. 뭔가 이상하더라구요..
희주님이 만드신 포스트 페이지를 봤는데... 음원이.. 카드에서 나오면 안되잖아요 지금..?
플레이바에서 음악이 나와야 하는건데 왜 저희 다 카드에서 음악을 재생하고 있었을까요..? 하하..

## 그래서 새벽에 급하게 플레이바랑 재생목록을 만들어왔습니다...
![image](https://github.com/user-attachments/assets/3761aa7d-d7c2-4e4c-91be-aad58032a30c)
놀랍게도 저기 우측에 반복 버튼 보이시죠..? 그게 재생목록 토글 버튼이고 누르면 재생목록이 나옵니다. 근데 왜 아이콘을 저걸로 했냐구요? 적당한 아이콘이 없어서 피그마에서 얼렁뚱땅 아이콘을 갖다 좀 썼어요.. 양해해주세요.. 그만큼 마음이 급했다는 뜻...

# 그래서 지금 구상한게
![image](https://github.com/user-attachments/assets/96a8e539-cbf2-4983-a2ad-16a10d7c68c4)
## 이런 가로형 트랙카드 컴포넌트를 만들어서 재생목록에 리스트 형태로 띄우도록 하고 그 컴포넌트에 음원 정보를 넣는걸로 해야할 것 같아요.
![image](https://github.com/user-attachments/assets/bd504fc2-a344-4958-9775-d4582b531ecd)
이렇ㄱ게...

# 정리하자면
1. 카드 컴포넌트 클릭하면
2. 가로형 트랙카드에 음원 정보 전송
3. 가로형 트랙카드는 플레이리스트에 추가됨

이렇게 구상했거든요
혹시 가능하다고 보십니까?ㅋㅋㅋㅋㅋㅋㅋ
제 미흡한 개발지식으로는.. 이 방법을 생각했는데 다른 좋은 방법이 있을까요...

일단 확실한건
저희가 지금까지 큐레이션, 아티스트, 포스트 카드 컴포넌트 만든거 있잖아요? 누르면 재생되도록 해놓았는데, **이걸 누르면 재생목록에 정보를 넣는걸로 기능을 바꿔야 해요**

여러분들의 의견을 좀 모아야 합니다.. 수정하는데 시간이 얼마나 걸릴지도 의논해봐야 하구요

**Comment by hjlee2778** [comment]
새벽에 재생플레이어 만드시느라 고생 많으셨습니다.. 우선 그거 완성되시면 제가 가져다가 바로 가능한지 테스트 해보겠습니다..!
